examples/list_slaves.rs

```diff
@@ -0,0 +1,33 @@
+//! EtherCAT master example: List slaves
+//!
+//! Opens master 0 (read-only), queries slave info for positions 0..31 and
+//! prints discovered slaves.
+use ethercat::{Master, SlavePos};
+use std::{io};
+
+pub fn main() -> Result<(), io::Error> {
+    env_logger::init();
+
+    // open the first available master interface
```

Comment:
I'm not sure if "first available" matches here. What happens if there are multiple (It's been so long since I last used it.)? Perhaps the meaning of “0” could be explained even better?